pvacseq/lib/generate_fasta.py

```diff
@@ -112,7 +113,10 @@ def main(args_input = sys.argv[1:]):
 
         if variant_type == 'FS':
             wildtype_subsequence, mutant_subsequence = get_frameshift_subsequences(position, full_wildtype_sequence, peptide_sequence_length, line)
-            mutant_subsequence += line['downstream_amino_acid_sequence']
+            downstream_sequence = line['downstream_amino_acid_sequence']
+            if args.downstream_sequence_length and len(downstream_sequence) > int(args.downstream_sequence_length):
```

Comment:
Having to convert the same value to an `int` twice in two lines seems a bit weird, but I'm not sure that introducing a temp variable makes it any better. If we're converting from a string to either `None` or a string for which `isdigit` is true upstream of this, can we do the conversion to an `int` there. Then the condition can be something like

`if args.downstream_sequence_length and len(downstream_sequence) > args.downstream_sequence_length` 

Either way. A really small gripe!